chapters/memory.txt

```diff
@@ -3096,18 +3096,168 @@ ifdef::VK_KHR_device_group[]
   * [[VUID-vkMapMemory-memory-00683]]
     pname:memory must: not have been allocated with multiple instances
 endif::VK_KHR_device_group[]
+ifdef::VK_EXT_map_memory_placed[]
+  * ename:VK_MEMORY_MAP_PLACED_BIT_EXT must: not be set in pname:flags
+endif::VK_EXT_map_memory_placed[]
 ****
 
 include::{generated}/validity/protos/vkMapMemory.txt[]
 --
 
-[open,refpage='VkMemoryMapFlags',desc='Reserved for future use',type='flags']
+ifdef::VK_EXT_map_memory_placed[]
+[open,refpage='VkMemoryMapFlagBits',desc='Bitmask specifying additional parameters of a memory map',type='enums']
+--
+Bits which can: be set in flink:vkMapMemory::pname:flags, specifying
```

Comment:
Feels a bit weird to call out vkMapMemory when the only flag explicitly _cannot_ be used with it, maybe:

```suggestion
Bits which can: be set in flink:vkMapMemory::pname:flags and flink:vkMapMemory2EXT::pname:flags, specifying
```

Reply:
Done.

Reply:
N.b. it's helpful to have back-references to all the users of a type, but it's fragile against extensions, and can get wordy. In the refpages it automatically generates "See Also" references for all APIs which use the API being described but we don't do that in the spec itself (though we could).